fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PyUtils.java

```diff
@@ -54,7 +54,7 @@
 	 */
 	//===========================================================
 	static String deviceImpl() {
-		return "PyTango.Device_4Impl";
+		return "PyTango.LatestDeviceImpl";
```

Comment:
Can we change all these `PyTango.` strings to `tango.`?  That is the preferred usage in PyTango.  Also note that the high level code for the device, e.g. https://github.com/tango-controls/pogo/blob/master/fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend, is now using `import tango`.
Although, I see the non-HL code is still using `import PyTango`, so it should also be updated.